### DISCUSSION TITLE
第5节课作业(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#303] [作业]
要记得群里的打分表也要填哟

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/677605262

**Comment by zhanghui-china** [comment]
[https://zhuanlan.zhihu.com/p/677639952](url)

**Comment by qianlian-mozi** [comment]
https://github.com/qianlian-mozi/internlm_practising_20240106/tree/main/lesson_5_homework

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/5.work.md

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135582525

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%BA%94%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%BA%94%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/homework/5/%E7%AC%AC5%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/5%E3%80%81LMDeploy-%E7%9A%84%E9%87%8F%E5%8C%96%E5%92%8C%E9%83%A8%E7%BD%B2%E8%AF%BE%E7%A8%8B-%E4%BD%9C%E4%B8%9A

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/3be50e96-48a5-40e2-9c51-a08ab1c22ed3
【FlowUs 息流】lmdeploy

**Comment by Everfighting** [comment]
https://github.com/Everfighting/InternLM_learning/blob/main/chapter5/chapter_5.md

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/677972589?

**Comment by codefd** [comment]
https://blog.csdn.net/googteo/article/details/135618764?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135618764%22%2C%22source%22%3A%22googteo%22%7D

**Comment by JustLike1** [comment]
https://github.com/InternLM/tutorial/discussions/303

**Comment by keesh0410** [comment]
https://zhuanlan.zhihu.com/p/679109211
